entity-framework/core/get-started/netcore/new-db-sqlite.md

```diff
@@ -85,13 +84,12 @@ The *blogging.db** SQLite DB is in the project directory.
 
 ### Changing the model:
 
-- If you make changes to your model, you can use the `dotnet ef migrations add` command to scaffold a new [migration](https://docs.microsoft.com/aspnet/core/data/ef-mvc/migrations#introduction-to-migrations)  to make the corresponding schema changes to the database. Once you have checked the scaffolded code (and made any required changes), you can use the `dotnet ef database update` command to apply the changes to the database.
-- EF uses a `__EFMigrationsHistory` table in the database to keep track of which migrations have already been applied to the database.
-- SQLite does not support all migrations (schema changes) due to limitations in SQLite. See [SQLite Limitations](../../providers/sqlite/limitations.md). For new development, consider dropping the database and creating a new one rather than using migrations when your model changes.
+- If you make changes to your model, you can use the `dotnet ef migrations add` command to scaffold a new [migration](https://docs.microsoft.com/aspnet/core/data/ef-mvc/migrations#introduction-to-migrations). Once you have checked the scaffolded code (and made any required changes), you can use the `dotnet ef database update` command to apply the schema changes to the database.
+- EF Core uses a `__EFMigrationsHistory` table in the database to keep track of which migrations have already been applied to the database.
+- For some kinds of schema changes, EF Core Migrations doesn't automatically generate code. See [SQLite Limitations](../../providers/sqlite/limitations.md). For new development, consider dropping the database and creating a new one rather than using migrations when your model changes.
```

Comment:
@bricelam does this still convey what we want here? Is it that we don’t generate code or we can’t generate the SQL?

Reply:
@tdykstra we do generate the "code" (e.g. the migration with the right operations). We throw at SQL generation time, e.g. when you try to apply the migration or generate the SQL script.